frontend/src/components/Psychologist/Reimbursement/Bill.jsx

```diff
@@ -94,6 +83,15 @@ const Bill = () => {
     ];
   };
 
+  const getFilteredDates = dates => Object.keys(dates).filter(date => {
```

Comment:
:green_circle: 
```suggestion
  const getFilteredDates = valuesByDate => Object.keys(valuesByDate).filter(date => {
```
Ce ne sont pas des dates, c'est une map à ce niveau là et le nom `dates` peut porter à confusion